### ISSUE TITLE
Feature request: Bazaar item sales list

### ISSUE THREAD

**Issue by danny-vvv** [EdenServer/eden-web#76]
A player has contacted me requesting implementing a full list of bazaar items for sale across the server, similar to that seen on another well-known private server website. This would allow players to browse items currently for sale, without searching for a specific item.

---

@slassen @bkesecker I am not sure how we are handling requests/feedback, hence I have put this here. We should probably have some sort of feedback system, plus a system of signing off feature requests. I would not want to build a new feature without having it green-lit first.

**Comment by bkesecker** [comment]
Up-front, not keen on the feature because I think the SQL query would span across a lot of data, which could bog down the game database if enough players are using it.

For feature requests, whether your own or someone else, asking like this in an issue is fine with me.  It will document the request and decision, should someone else request similar.

**Comment by danielvarcas** [comment]
Yes that sounds fair. Perhaps it also is removing too much of a gameplay element, although that's purely subjective.

**Comment by InoUno** [comment]
There's already a caching layer implemented in the server code, which is used for some of the other queries. So we should be able to just reuse that to avoid many big DB queries.

**Comment by bkesecker** [comment]
> There's already a caching layer implemented in the server code, which is used for some of the other queries. So we should be able to just reuse that to avoid many big DB queries.

Does that work like this?  Search occurs for "ore" at time T, results cached, search for "ore" at time T+X, results updated?

**Comment by slassen** [comment]
I wrote it so that it works on a key. This can be the url query string or just a route, or whatever.
